Gemfile

```diff
@@ -6,6 +6,7 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 # Tools
 gem 'bootsnap', require: false
+gem 'cucumber', require: false, group: :test
```

Comment:
I see we already have a `:test` group but it really doesn't make much sense? _Everything_ in the `Gemfile` is only for testing.

Reply:
I was following the convention that we have for `simplecov`. I don't feel strongly about whether we use the `:test` group...so it's your call :)

Reply:
Let's remove them all then 💅